### PR TITLE
Add pricing section

### LINKED ISSUE #52: Create pricing tier section
Please see the Figma project for assets, specs, and most up-to-date mock-up.

image

### PR DESCRIPTION
**Pull request by sttic**
Resolves #52 

Channel URL (cusec-2021-staging): https://cusec-2021-staging--pr52-feat-pricing-section-l5r7r3a8.web.app [expires 2020-10-25 18:57:20 EST]

### CODE REVIEW

File: components/sections/Pricing.tsx [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+    "All free tier benefits",
+    "CUSEC 2021 merch",
+    "Exclusive and priority networking",
+    "Access to a private VIP communication platform with the sponsors and speakers",
+    "VIP Q&A events at the end of each night with speakers and sponsors",
+    "And more to come!",
+  ],
+};
+
+const monoGridArea = "I_LOVE_CUSEC";
```
Comment (gordonpn) [inline components/sections/Pricing.tsx:45]: haha

**Review by gordonpn — APPROVED** [review]
Looks amazing! 🎉

The Firebase channel feature is also super cool.

File: components/RegisterButton.tsx [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+import theme from "@/src/theme";
+
+export default function RegisterButton(): ReactElement {
+  return (
+    <PseudoBox
+      as="button"
+      height="42px"
+      transition="all 0.2s"
+      paddingX="42px"
+      rounded="9999px"
```
[the remark this answers is not in the document]
Comment (sttic) [reply components/RegisterButton.tsx:12]: I was lazy and didn't want to keep changing it when I was playing around with the height :joy:

File: components/sections/Pricing.tsx (hunk 1 above)
Comment (sttic) [reply components/sections/Pricing.tsx:45]: I thought this was more fun than something typical when choosing an arbitrary string for the grid layout :stuck_out_tongue:

**Comment by sttic** [comment]
> Looks amazing! tada
> 
> The Firebase channel feature is also super cool.

Thanks :heart: !

It'll be even cooler if the workflow/action for it worked and automatically has a GitHub bot post a link for it!!

File: components/sections/Pricing.tsx (hunk 1 above)
Comment (gordonpn) [reply components/sections/Pricing.tsx:45]: For sure!

**Pull request merged**